extensions/windows-event-log/Bookmark.cpp

```diff
@@ -77,18 +74,18 @@ Bookmark::Bookmark(const std::wstring& channel,
     state_manager_->set(state_map);
   }
 
-  if (!(hBookmark_ = unique_evt_handle{ EvtCreateBookmark(0) })) {
+  if (!(hBookmark_ = unique_evt_handle{ EvtCreateBookmark(nullptr) })) {
```

Comment:
at line 64, the assignment was separated from the if statement, should we separate it here as well?

Reply:
good idea, I've changed it in https://github.com/apache/nifi-minifi-cpp/pull/1434/files#diff-af3d1e50a2bd10e0b4ceda4fe27aaad8697ad7c82e8ffb99e188b20b2d7fff0cR64-R65